### ISSUE TITLE
Android Mobile App Security

### ISSUE THREAD

**Issue by jmelz13** [electroneum/electroneum#176]
Mobile mining appears to keep the android app unlocked on my phone while it is running in the background. When I reopen the app I am not required to re-enter my password.

**Comment by thedyerman** [comment]
I don't have this problem (Galaxy S8, Android 7.0), what android version / model phone are you running?

**Comment by jmelz13** [comment]
Pixel 1, Android 8.1

**Comment by Tadracket** [comment]
My phone does the same thing but I am not concerned. #1 - I only mine at night when my phone is at home connected to wireless and plugged into the charger. #2 - my phone is passcode protected.

**Comment by thedyerman** [comment]
@jmelz13 
I misread what you wrote, I interpreted that to mean it was keeping your phone unlocked. I get it now, You mean your Electroneum wallet did not require a pass code after opening the application and allowing it to run in the background. When you go back into Electroneum it did not require a code again. 

I did a little research and this is normal practice. Paypal for instance will not require you to re-pin when it is running in the background and you simply go back to it. There is a fine balance between security and convenience and it would be frustrating to users if every time they checked their mining they had to enter the pin to get into the phone then their other pin to get into the app (assuming it was already running in the background, I accidentally close mine all the time)
